src/NzbDrone.Core/DecisionEngine/Specifications/RawDiskSpecification.cs

```diff
@@ -21,24 +21,26 @@ public RawDiskSpecification(Logger logger)
 
         public RejectionType Type => RejectionType.Permanent;
 
-        public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase searchCriteria)
+        public virtual Decision IsSatisfiedBy(RemoteAlbum subject, SearchCriteriaBase searchCriteria)
         {
             if (subject.Release == null || subject.Release.Container.IsNullOrWhiteSpace())
             {
                 return Decision.Accept();
             }
 
-            if (_dvdContainerTypes.Contains(subject.Release.Container.ToLower()))
-            {
-                _logger.Debug("Release contains raw DVD, rejecting.");
-                return Decision.Reject("Raw DVD release");
-            }
+            // TODO Implement for Music Releases if applicable.
```

Comment:
Not sure that there is such a thing as a raw rip for music releases, but I left it in encase it pops up. Removing blu-ray types and changing DVD to CD. Modified test fixture accordingly.